addons/mail/static/src/discuss/call/common/call.js

```diff
@@ -109,17 +132,54 @@ export class Call extends Component {
     }
 
     get visibleMainCards() {
-        if (this.props.thread.activeRtcSession) {
+        const activeSession = this.props.thread.activeRtcSession;
+        if (activeSession) {
+            let videoType;
+            switch (activeSession.videoStreams.size) {
+                case 1:
+                    this.state.insetCard = false;
+                    if (
+                        activeSession.mainVideoStream !==
+                        activeSession.videoStreams.values().next().value
+                    ) {
+                        this.props.thread.activeRtcSession = false;
+                        activeSession.mainVideoStream = false;
+                    }
+                    break;
+                case 2:
+                    videoType =
```

Comment:
the variable only seems to be used there, so you can probably define it as a `const`